### PR TITLE
Tightly couple verify password and new password

### LINKED ISSUE #2049: tightly couple verify password and new password
**Actual Behaviour**

When something is entered in the fields and the removed, the validation text remains.

**Expected Behaviour**

The validation text should not be displayed when the fields are empty.

**Steps to reproduce it**

1. Settings -> Password
2. Type something in any of the 3 inputs requested.
3. Clear the input.

**Screenshots of the issue**

issue7

**Would you like to work on the issue?**

Yes

### PR DESCRIPTION
**Pull request by ArexJambusarwala**
Fixes #2049 

Changes: Changes in file ChangePassword.react.js are made which when input is cleared no error message is shown.

Demo Link : https://pr-2053-fossasia-susi-web-chat.surge.sh/

Screenshots of the change: 

corrected6

### CODE REVIEW

**Comment by ArexJambusarwala** [comment]
If the error message is supposed to be shown, that it has some inconsistencies. For example, if we type something in verify password and remove it, the error message is displayed. Now if we type something in "new password" field and remove it,  then error message of "verify password" goes away.(sice both fields now have same value)

**Comment by AakashMallik** [comment]
> If the error message is supposed to be shown, that it has some inconsistencies. For example, if we type something in verify password and remove it, the error message is displayed. Now if we type something in "new password" field and remove it, then error message of "verify password" goes away.(since both fields now have same value). I will fix the unsuccessful check if this change is indeed required.

The verify password issue is a legit case. I had raised an issue regarding this but that was done for the signup component. You can follow along with my old PR and make similar changes here. https://github.com/fossasia/chat.susi.ai/pull/1767
I coupled the verify password and password field together to address this problem, but it a **similar** problem but from a different **perspective**. Let me know if you need further clarification.

**Comment by ArexJambusarwala** [comment]
I hope this is what is required:
http://pr-2053-fossasia-susi-web-chat.surge.sh/

Verify password error message remains unlike before.

**Comment by AakashMallik** [comment]
CI failing.. please check plus files changed turn up to be zero...

**Comment by ArexJambusarwala** [comment]
The Cl test fails stating:
   Aborted - you do not have permission to publish to pr-2053-fossasia-susi-web-chat.surge.sh
How to go through with this?

File: src/components/Auth/ChangePassword/ChangePassword.react.js [hunk 1]
```diff
@@ -166,8 +166,9 @@ class ChangePassword extends Component {
           : '';
 
         const newPasswordConfirmError =
-          (confirmNewPassword || newPasswordConfirmErrorMessage) &&
-          !(confirmNewPassword === newPassword);
+          ((confirmNewPassword || newPasswordConfirmErrorMessage) &&
+            !(confirmNewPassword === newPassword)) ||
+          (newPassword === '' && newPasswordConfirmErrorMessage);
```
Comment (AakashMallik) [inline src/components/Auth/ChangePassword/ChangePassword.react.js]: replace ```newPassword === ''``` with ```!newPassword```

**Comment by fragm3** [comment]
Closing, PR not needed

**Pull request closed without merging**